packages/xo-server/src/xo-mixins/backups.js

```diff
@@ -374,7 +376,7 @@ export default class {
         bypassVdiChainsCheck: force,
         snapshotNameLabel: `XO_DELTA_EXPORT: ${targetSr.name_label} (${targetSr.uuid})`,
       })
-      $defer.onFailure(() => srcXapi.deleteVm(delta.vm.uuid))
+      $defer.onFailure(() => srcXapi.VM_destroy(delta.vm.$ref))
```

Comment:
Unfortunately, this is not working, `delta.vm` only contains XAPI properties of a VM, not special ones added by `xen-api`, (those prefixed with `$`).

You should be able to use `getXapiObject()`.